aws_lambda_powertools/utilities/parser/models/apigwv2.py

```diff
@@ -25,7 +25,7 @@ class RequestContextV2AuthorizerIam(BaseModel):
 
 class RequestContextV2AuthorizerJwt(BaseModel):
     claims: Dict[str, Any]
-    scopes: List[str]
+    scopes: Optional[List[str]] = None
```

Comment:
can we have a quick test to avoid future regression?

We can take this event, remove the `scopes` dict key and run the model - no need to assert is `None`, if the Model works without it, it means validation worked as expected.

https://github.com/aws-powertools/powertools-lambda-python/blob/bfde1c9124910851f3de41b1db54e48e2bba7f16/tests/unit/parser/test_apigwv2.py#L41

Reply:
I would like to keep this test and this field to validate that we are parsing as a List when a value is provided.

My thoughts:

1 - I can add more lines in this test to remove this field and run the model again
```python
    .
    .
    .
    assert parsed_event.stageVariables == raw_event["stageVariables"]

    # Removing scope field
    raw_event["requestContext"]["authorizer"]["jwt"]["scopes"] = None
    parsed_event: APIGatewayProxyEventV2Model = APIGatewayProxyEventV2Model(**raw_event)
```

2 - I can create another test to test specific this, but I think this is too much in my opinion.

What do you suggest?

Reply:
Create another test specific to the claim being none. It'll be only a few lines, and if it fails we know what contract it broke.

I'll post a sample what I mean by that after a meeting

Reply:
Thanks for creating the test @heitorlessa! I created a new one before you send this haha! :heart: